golang/cosmos/app/params/params.go

```diff
@@ -5,3 +5,5 @@ const (
 	StakePerAccount           = "stake_per_account"
 	InitiallyBondedValidators = "initially_bonded_validators"
 )
+
+const MaxBlockMaxBytes = 5242880 // 5MB
```

Comment:
Rename to match the Tendermint/CometBFT variable that this is supplanting, and refer back to it. Also make the number less magical by showing how it's computed.

```suggestion
// MaxBlockSizeBytes is the maximum permitted size of the blocks.
// See github.com/cometbft/cometbft/types.MaxBlockSizeBytes.
// We implement our own limit here to avoid modifying the consensus library.
const MaxBlockSizeBytes = 5 * 1024 * 1024 // 5MB
```

Reply:
For the value, this is a copy paste of the currently proposed value in the 1518 PR against CometBFT: https://github.com/cometbft/cometbft/pull/1518/commits/4c373ebbfbe099dcedae5024255d119cdfa0235f

Reply:
Also to be clear, I'd like this value to be the default max block size param value of a new chain, and the new value the param is updated to during the upgrade, not the new absolute max value allowed by the app. If for some reason we realize the value is too low, governance should still be able to increase the value without resorting to a software upgrade.

Reply:
SGTM.